modules/ROOT/pages/dialog-components.adoc

```diff
@@ -330,6 +330,7 @@ NOTE: To replace the entire dialog body with an iframe that loads its content fr
 |name |string |required |A identifier for the button.
 |label |string |optional |String to use for the iframe's `+title+` attribute.
 |sandboxed |boolean |optional |default: `+true+` - When true, sets the iframe's `+sandbox+` attribute to `+"allow-scripts allow-same-origin"+`.
+|transparent |boolean |optional |default: `+true+` - When true, sets the iframe's background to transparent instead of opaque.
```

Comment:
This needs the `NOTE: This feature is only available for TinyMCE 6.1 and later.` warning as well I think?

Reply:
Yeah, I was just working on adding it 👍

Reply:
Done

Reply:
This should be using the partial shouldn't it?

Reply:
Yep, fixed.